### ISSUE TITLE
Cant install plotly-orca, connection failed

### ISSUE THREAD

**Issue by EdvinAblad** [sglyon/ORCA.jl#26]
I don't think this is a duplicate of other issues (even though quite similar).

It seem that: `https://conda.anaconda.org/plotly/win-64/current_repodata.json` don't exist, can't find it using a browser... And thus plotly-orca fails to install and thus also `]build ORCA`

For your knowledge I have the following system
```
Julia Version 1.4.0
Commit b8e9a9ecc6 (2020-03-21 16:36 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-8.0.1 (ORCJIT, skylake)
```
I have isolated the error from julia by replicating the error using the following code in cmd 
```
conda.exe install -y plotly-orca
Collecting package metadata (current_repodata.json): failed

CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/plotly/win-64/current_repodata.json>
Elapsed: -

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.
'https://conda.anaconda.org/plotly/win-64'
```
The original error message from julia is:
```
(@v1.4) pkg> build ORCA
   Building Conda → `C:\Users\ablad\.julia\packages\Conda\3rPhK\deps\build.log`
   Building ORCA ─→ `C:\Users\ablad\.julia\packages\ORCA\fiEmb\deps\build.log`
┌ Error: Error building `ORCA`: 
│ Warning: 'plotly' already in 'channels' list, moving to the top
│ Collecting package metadata (current_repodata.json): ...working... failed
│ 
│ CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/plotly/win-64/current_repodata.json>
│ Elapsed: -
│ 
│ An HTTP error occurred when trying to retrieve this URL.
│ HTTP errors are often intermittent, and a simple retry will get you on your way.
│ 'https://conda.anaconda.org/plotly/win-64'
│ 
│ 
│ [ Info: Running `conda config --add channels plotly --file 'C:\Users\ablad\.julia\conda\3\envs\_ORCA_jl_\condarc-julia.yml' --force` in _ORCA_jl_ environment
│ [ Info: Running `conda install -y plotly-orca` in _ORCA_jl_ environment
│ ERROR: LoadError: failed process: Process(setenv(`'C:\Users\ablad\.julia\conda\3\Scripts\conda.exe' install -y plotly-orca`,["PATH=C:\\Users\\ablad\\.julia\\conda\\3\\envs\\_ORCA_jl_\\Library\\bin;C:\\gurobi901\\win64\\bin;C:\\Program Files (x86)\\Common Files\\Oracle\\Java\\javapath;C:\\WINDOWS\\system32;C:\\WINDOWS;C:\\WINDOWS\\System32\\Wbem;C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\;C:\\WINDOWS\\System32\\OpenSSH\\;C:\\ProgramData\\chocolatey\\bin;C:\\Program Files\\Git\\cmd;C:\\Program Files\\Intel\\WiFi\\bin\\;C:\\Program Files\\Common Files\\Intel\\WirelessCommon\\;C:\\Users\\ablad\\AppData\\Local\\Microsoft\\WindowsApps;C:\\Users\\ablad\\AppData\\Local\\Programs\\Microsoft VS Code\\bin;C:\\Users\\ablad\\AppData\\Local\\Programs\\MiKTeX 2.9\\miktex\\bin\\x64\\;C:\\Users\\ablad\\AppData\\Local\\atom\\bin", "USERDOMAIN_ROAMINGPROFILE=MIMON21", "HOMEPATH=\\Users\\ablad", "PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC", "SESSIONNAME=Console", "GKS_USE_CAIRO_PNG=true", "APPDATA=C:\\Users\\ablad\\AppData\\Roaming", "PSMODULEPATH=C:\\Program Files\\WindowsPowerShell\\Modules;C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules", "SYSTEMROOT=C:\\WINDOWS", "COMMONPROGRAMW6432=C:\\Program Files\\Common Files"  …  "PROGRAMFILES(X86)=C:\\Program Files (x86)", "PROGRAMFILES=C:\\Program Files", "GKS_FONTPATH=C:\\Users\\ablad\\.julia\\packages\\GR\\yMV3y\\src\\..\\deps\\gr", "LOGONSERVER=\\\\MIMON21", "DRIVERDATA=C:\\Windows\\System32\\Drivers\\DriverData", "CONDA_PREFIX=C:\\Users\\ablad\\.julia\\conda\\3\\envs\\_ORCA_jl_", "JULIA_LOAD_PATH=@;C:\\Users\\ablad\\AppData\\Local\\Temp\\jl_KIoddH", "SYSTEMDRIVE=C:", "PROCESSOR_ARCHITECTURE=AMD64", "OPENBLAS_MAIN_FREE=1"]), ProcessExited(1)) [1]
│
│ Stacktrace:
│  [1] pipeline_error at .\process.jl:525 [inlined]
│  [2] run(::Cmd; wait::Bool) at .\process.jl:440
│  [3] run at .\process.jl:438 [inlined]
│  [4] runconda(::Cmd, ::Symbol) at C:\Users\ablad\.julia\packages\Conda\3rPhK\src\Conda.jl:114
│  [5] add(::String, ::Symbol; channel::String) at C:\Users\ablad\.julia\packages\Conda\3rPhK\src\Conda.jl:188
│  [6] add(::String, ::Symbol) at C:\Users\ablad\.julia\packages\Conda\3rPhK\src\Conda.jl:187
│  [7] top-level scope at C:\Users\ablad\.julia\packages\ORCA\fiEmb\deps\build.jl:5
│  [8] include(::String) at .\client.jl:439
│  [9] top-level scope at none:5
│ in expression starting at C:\Users\ablad\.julia\packages\ORCA\fiEmb\deps\build.jl:5
└ @ Pkg.Operations D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.4\Pkg\src\Operations.jl:892
```

**Comment by EdvinAblad** [comment]
Found a work around: use a pre installed ([Conda](https://github.com/JuliaPy/Conda.jl#using-a-pre-existing-conda-installation))
Create the _ORCA_jl_ environment mannually from conda
Install plotly-orca with Conda.add

**Comment by sglyon** [comment]
Thank you for posting the fix

**Comment by sglyon** [comment]
@EdvinAblad would you be willing to paste a snippet of code that creates the conda environment (in the right place) and installs orca? 

Other users are hitting this issue and we'd like to help them out too.

Thanks!

**Comment by EdvinAblad** [comment]
I didn't post an exact solution since I basically tested things until it worked... But I try my best to give some more details on what I think did the trick... (This is also the first time I use Conda so I don't really know what I'm doing)

So I installed miniconda3 for windows and hooked it up with Julia [Conda](https://github.com/JuliaPy/Conda.jl#using-a-pre-existing-conda-installation) 

Then for every failed package I tried to decipher which package that  was missing and in what environment.

I then created an environment matching the one created by Julia. 
Don't recall if I found a path: `conda create -p PATH` 
Or if I sufficed to create a with the same name `conda create -n _ORCA_jl_`

(You might need to, e.g., initialize the environment with default packages, but don't recall how I did it. In the end it looked like the base conda environment (maybe a copy of it?))

When the environment is in-place I added the package from the Julia repl
`Conda.add("plotly-orca","C:\\Users\\ablad\\miniconda3\\envs\\conda_jl\\envs\\_ORCA_jl_",channel="plotly")`  (Julia repl have better memory then me)

I'm truly sorry for not being able to give a better description of what I did to solve this.
(Don't have the time and I don't dare to mess around to replicate it)
Hopefully it will be of some help anyway...
